applications/new_schema/types.py

```diff
@@ -87,3 +93,62 @@ def resolve_harbor_choices(self, info, **kwargs):
     @view_permission_required(BerthApplication, BerthLease, CustomerProfile)
     def get_node(cls, info, id):
         return super().get_node(info, id)
+
+
+WinterStorageMethodEnum = graphene_enum(WinterStorageMethod)
```

Comment:
nitpick: I would keep all enum declarations in the beginning of the file together, as they are one-liners that might easily get lost between bigger blocks of code. makes it easier to navigate things in github's web UI, if you need to look something up

Reply:
ok! will move it